tests/python2-libraries/requirements.txt

```diff
@@ -139,53 +138,53 @@ pymongo==3.4.0
 pymysql==0.7.11
 pyopenssl==17.0.0
 pyparsing==2.2.0
-pyramid==1.8.3
+pyramid==1.8.4
 pystache==0.5.4
 pytest-cov==2.5.1
-pytest==3.0.7
+pytest==3.1.2
 python-cjson==1.2.1
 python-daemon==2.1.2
 python-dateutil==2.6.0
 python-gflags==3.1.1
-python-keystoneclient==3.10.0
+python-keystoneclient==3.11.0
 python-memcached==1.58
 python-mimeparse==1.6.0
-python-novaclient==8.0.0
+python-novaclient==9.0.1
 python-subunit==1.2.0
 python-swiftclient==3.3.0
 pytz==2017.2
 pyyaml==3.12
 pyzmq==16.0.2
-raven==6.0.0
+raven==6.1.0
 redis==2.10.5
 repoze.lru==0.6
 requests-oauthlib==0.8.0
-requests==2.14.2
+requests==2.18.1
 retrying==1.3.3
 rsa==3.4.2
 scipy==0.19.0
-selenium==3.4.1
+selenium==3.4.3
 setuptools-git==1.2
-setuptools==35.0.2
-sh==1.12.13
-simplejson==3.10.0
+setuptools==36.0.1
+sh==1.12.14
+simplejson==3.11.0
```

Comment:
Not sure what happened (PyPI mirror issue), because I wasn't able to find 3.11.0 until a few minutes ago, but it did appear in some Google search results (but the links went to 3.10.0 or were 404's)